src/server/podcastRssFeed.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import config from 'config';
+import { fetchSeries } from '../util/audioApi';
+
+const podcastRssFeed = async (seriesId: number): Promise<string> => {
+  try {
+    const series = await fetchSeries(seriesId, 'nb');
+    const podcastUrl = `${config?.ndlaFrontendDomain}/podcast/${series.id}`;
+    const ownerEmail = 'support+podcast@ndla.no';
```

Comment:
Lurt!

Reply:
Jeg så vi hadde gjort tilsvarende for api'ene, så da regner jeg med at det var en grei løsning her og :smile: